server/routes/contacts/manage/start/startManageContactsJourneyController.ts

```diff
@@ -0,0 +1,34 @@
+import { Request, Response } from 'express'
+import { v4 as uuidv4 } from 'uuid'
+import { Page } from '../../../../services/auditService'
+import { PageHandler } from '../../../../interfaces/pageHandler'
+import ManageContactsJourney = journeys.ManageContactsJourney
+
+export default class StartManageContactsJourneyController implements PageHandler {
+  public PAGE_NAME = Page.MANAGE_CONTACTS_START_PAGE
+
+  private MAX_JOURNEYS = 5
+
+  GET = async (req: Request, res: Response): Promise<void> => {
+    // Create a new journey object for managing contacts
+    const journey: ManageContactsJourney = { id: uuidv4(), lastTouched: new Date() }
+
+    // Create the object for journeys if not present in the session
+    if (!req.session.manageContactsJourneys) {
+      req.session.manageContactsJourneys = {}
+    }
+
+    // Add this journey to the list
+    req.session.manageContactsJourneys[journey.id] = journey
+
+    // Replace the oldest journey if more than MAX_JOURNEYS exist
+    if (Object.entries(req.session.manageContactsJourneys).length > this.MAX_JOURNEYS) {
```

Comment:
This seems like the sort of thing we could make generic and then immediately regret as we discover intricacies in each journey. What do you think?

Reply:
It definitely feels repetitive, and I borrowed most of this from the create journey. However, it is effectively working on different types/structures - unless we make them inherit a superclass containing this data/logic for max journeys?

Reply:
🤷  I say we just C&P for now. We might be able to make some utility functions to help make it more concise. I'll have a think (easy in Kotlin 😂 )